__tests__/e2e/scripts/test.sh

```diff
@@ -23,5 +23,13 @@ echo "✅ Setup completed successfully. Running e2e tests..."
 #
 # run e2e tests journey
 ##################################################
+set +e
+npx @elastic/synthetics uptime.journey.ts --reporter junit | tee tmp/reporter.out
+errorLevel=$?
 
-npx @elastic/synthetics uptime.journey.ts
+#
+# transform reporter to junit only format
+##################################################
+grep -v "Waiting" tmp/reporter.out > tmp/junit.xml
```

Comment:
There are some console.log references for the waiting for the services to be up and running, those traces cause issues since the junit reporter uses the stdout

Reply:
Let me add support for outputting the data in a specific directory or a file. 

Btw, you can also pass a custom std stream to the Synthetic agent through `--outfd <fd>` option.

Reply:
```sh
SYNTHETICS_JUNIT_FILE='/path/to/junit.xml' npx @elastic/synthetics suites --reporter junit
```

added support for extracting the xml to a file.

Reply:
I'll then change it, thanks for the quick answer :)